README.md

```diff
@@ -205,29 +152,20 @@ Example:
 
 > Mention all of the collaborators of this project.
```

Comment:
- [ ] Please delete all lines in the README that start with `>`, like line 153. These lines are just section guides telling you how to fill in each section. Remember to delete them once the section is filled in.